### PR TITLE
Addon UI

### PR DESCRIPTION
**Pull request by rehandalal**
f?

### CODE REVIEW

**Comment by rehandalal** [comment]
r?

**Comment by rehandalal** [comment]
N.B.: The only way to sign in currently is to go to `/admin/`.

File: morgoth/base/static/js/components/AddonForm.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+const style = {
+  toolbar: {
+    justifyContent: 'flex-end',
+  },
+  refresh: {
+    display: 'inline-block',
+    position: 'relative',
+  },
+};
```
Comment (brittanystoroz) [inline morgoth/base/static/js/components/AddonForm.jsx]: ahhhh this is so weird to me still but everyone seems to enjoy doing it these days

**Comment by brittanystoroz** [comment]
This looks good to me, I would maybe start early with adding tests, but clearly I am not a stickler for those. @mythmon did you want to take a look at this as well? Otherwise r+

File: morgoth/base/static/js/components/pages/HomePage.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
 import Paper from 'material-ui/Paper';
-import ActionExtension from 'material-ui/svg-icons/action/extension';
-import NavigationApps from 'material-ui/svg-icons/navigation/apps';
+import ActionExtension from '../../../../../../node_modules/material-ui/svg-icons/action/extension';
+import NavigationApps from '../../../../../../node_modules/material-ui/svg-icons/navigation/apps';
```
Comment (mythmon) [inline morgoth/base/static/js/components/pages/HomePage.jsx]: Couldn't you do this as `import ActionExtension from 'material-ui/svg-icons/action/extension';`?

Comment (rehandalal) [inline morgoth/base/static/js/components/pages/HomePage.jsx]: gah! yep. i think my IDE rewrote this, when I moved the files.

Comment (rehandalal) [inline morgoth/base/static/js/components/pages/HomePage.jsx]: stupid "smart refactoring"

**Pull request merged**